src/flatten.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Dash;
+
+/**
+ * Gets a list of nested elements in `$iterable`.
+ *
+ * Keys in `$iterable` are not preserved.
+ *
+ * @see groupBy()
```

Comment:
Is it in fact related to groupBy?

Reply:
In a way, it is the inverse of group by.